diff --git a/com.crm.Training/src/test/java/com/maventraining/Checkgittest.java b/com.crm.Training/src/test/java/com/maventraining/Checkgittest.java
new file mode 100644
index 0000000..bf08d9d
--- /dev/null
+++ b/com.crm.Training/src/test/java/com/maventraining/Checkgittest.java
@@ -0,0 +1,21 @@
+package com.maventraining;
+
+import org.testng.annotations.Test;
+
+public class Checkgittest {
+	@Test
+	public void learn() {
+		System.out.println("maven learning");
+	}
+	@Test
+	public void Read() {
+		System.out.println("maven reading");
+	}
+
+	@Test
+	public void Write() {
+		System.out.println("maven Writing");
+	}
+	}
+
+
